src/plugins/ui_actions/public/types.ts

```diff
@@ -57,10 +57,12 @@ export interface TriggerContextMapping {
 const DEFAULT_ACTION = '';
 export const ACTION_VISUALIZE_FIELD = 'ACTION_VISUALIZE_FIELD';
 export const ACTION_VISUALIZE_GEO_FIELD = 'ACTION_VISUALIZE_GEO_FIELD';
+export const ACTION_VISUALIZE_LENS_FIELD = 'ACTION_VISUALIZE_LENS_FIELD';
 export type ActionType = keyof ActionContextMapping;
 
 export interface ActionContextMapping {
   [DEFAULT_ACTION]: BaseContext;
   [ACTION_VISUALIZE_FIELD]: VisualizeFieldContext;
   [ACTION_VISUALIZE_GEO_FIELD]: VisualizeFieldContext;
+  [ACTION_VISUALIZE_LENS_FIELD]: VisualizeFieldContext;
```

Comment:
I missed this mapping in previous pr, 
Previously we kept such plugin specific definition inside the plugins that own them and extended this mapping from outside. 
example: https://github.com/elastic/kibana/blob/master/src/plugins/dashboard/public/plugin.tsx#L172

it also isn't ideal, but makes a bit more sense.

Reply:
I had it like that on the previous PR but Tim suggested to not use this declare module, so this is the reason we added it there